src/test/java/org/redischool/attendance/FeatureTestForSecurity.java

```diff
@@ -24,7 +24,7 @@
                 "credentials.password=bar"
         }
 )
-@RunWith(SpringJUnit4ClassRunner.class)
+@ExtendWith(SpringExtension.class)
```

Comment:
It is already included in the @SpringBootTest, can be deleted

Reply:
Fixed.